pkgs/by-name/jo/jol-cli/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gitUpdater,
+  jre_headless,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

Ah, since you also don't compile any C, better use
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

---

Thanks, changed to noCC